parser/prism/Parser.cc

```diff
@@ -11,7 +12,7 @@ unique_ptr<parser::Node> Parser::run(core::GlobalState &gs, core::FileRef file)
     Prism::Parser parser{source};
     Prism::ParseResult parseResult = parser.parse();
 
-    return Prism::Translator(parser, gs, file).translate(move(parseResult));
+    return Prism::Translator(parser, gs, file, parseResult.parseErrors).translate(parseResult.getRawNodePointer());
```

Comment:
For a future change, it might be nice to replace this pattern with a single static method on `Translator`.